tests/core/config/__init__.py

```diff
@@ -0,0 +1,19 @@
+from symphony.bdk.core.config.model.bdk_retry_config import BdkRetryConfig
+
+
+def minimal_retry_config():
```

Comment:
something worth checking maybe: we should make sure that all tests are still fast to run (default retry configurations were causing some tests to be slow because of the wait between retries for the Java BDK)

Reply:
i.e are the config files below with 1s sleep affecting the test executions?

Reply:
The configuration files were used only for testing the configuration part (they are not used in other tests)
The methods below returns BdkRetryConfig should use a wait of 10ms